Tiltfile.dc

```diff
@@ -1,17 +1,22 @@
 # -*- mode: Python -*-
-enable_feature("team_alerts")
+enable_feature("snapshots")
 
 docker_compose('docker-compose.yml')
 
+# if docker-compose.yml specifies an `Image`, Tilt will auto-associate
+# a docker_build of the same name with that service.
 docker_build('vigoda_for_dc', 'vigoda', dockerfile='vigoda/Dockerfile.dc')
-dc_resource('vigoda', image='vigoda_for_dc')
 
-docker_build('fortune_for_dc', 'fortune', dockerfile='fortune/Dockerfile.dc',
+docker_build('servantes_fortune', 'fortune', dockerfile='fortune/Dockerfile.dc',
   live_update = [
     sync('fortune', '/go/src/github.com/windmilleng/servantes/fortune'),
     run('cd src/github.com/windmilleng/servantes/fortune && make proto'),
     run('go install github.com/windmilleng/servantes/fortune'),
     restart_container(),
   ])
 
-dc_resource('fortune', image='fortune_for_dc')
+# if docker-compose.yml doesn't explicitly set `Image` for a service,
+# you can use `dc_resource` to configure the docker_build <--> DC service
+# linkage. NOTE: image name must be in the form expected by Docker Compose,
```

Comment:
i don't understand this note? AFAIK, Docker Compose has no such requirement

Reply:
so if in dc.yml you don't specify an `image`, DC will build/attempt to run an image named `<dir>_<service>`.

Tilt has no way of overwriting the image that DC uses, so the image that we docker_build has to match the one that DC attempts to run. (if we `docker_build` "servantes_vigoda" then DC will correctly start a container using the image we built; if we name the image anything else, DC will look for "servantes_vigoda", not find it, and error)